_maps/shuttles/pirate_default.dmm

```diff
@@ -534,11 +534,11 @@
 /obj/effect/turf_decal/stripes/line,
 /obj/effect/decal/cleanable/dirt,
 /obj/structure/table,
-/obj/item/gun/energy/laser{
+/obj/item/gun/energy/laser/rifle{
```

Comment:
Why do pirates have cheap mass-produced Nanotrasen knock offs rather than the real ones? This goes for the Thunderdome and ruins ones as well.

Reply:
i mean, they had it before, I didn't change nothing here. rifle is just the subtype for bulky lasers